stackstorm.yml

```diff
@@ -3,12 +3,17 @@
   hosts: all
   environment: "{{ st2_proxy_env | default({}) }}"
   roles:
-    - StackStorm.mongodb
+    - role: StackStorm.mongodb
+      mongodb_version: 4.0
+      mongodb_apt_keys:
+        "3.2": "42F3E95A2C4F08279C4960ADD68FA50FEA312927"
+        "3.4": "0C49F3730359A14518585931BC711F9BA15703C6"
+        "4.0": "9da31620334bd75d9dcb49f368818c72e52529d4"
     - StackStorm.rabbitmq
     - StackStorm.postgresql
     - StackStorm.st2repo
     - StackStorm.st2
-    - StackStorm.st2mistral
+    # - StackStorm.st2mistral
```

Comment:
Good start. Now need to add condition to disable this role import for Ubuntu Bionic only.

Also keep in mind that the same condition should be applied to `postgresql` role as well as that's used by mistral only.